packages/@o3r/core/schematics/index.it.spec.ts

```diff
@@ -23,7 +23,7 @@ describe('new otter application', () => {
     execAppOptions.cwd = appFolderPath;
   });
   test('should build empty app', () => {
-    packageManagerExec(`ng add --skip-confirmation @o3r/core@${o3rVersion} --preset=cms`, execAppOptions);
+    packageManagerExec(`ng add --skip-confirmation @o3r/core@${o3rVersion} --preset=cms --enable-analytics`, execAppOptions);
```

Comment:
it's weird to add a deprecated flag to prevent the test from failing :)

Reply:
There is an issue in the component creation that does not support the `--enable-analytics=false`.
But the fix of the component should not be part of this PR.

Reply:
we actually want to test that the component generator works with analytics
I think we should manually add @o3r/analytics instead of using the deprecated flag
```suggestion
    packageManagerExec(`ng add --skip-confirmation @o3r/core@${o3rVersion} --preset=cms`, 
    packageManagerExec(`ng add --skip-confirmation @o3r/analytics@${o3rVersion}`, execAppOptions);
```